### PR TITLE
5378: extends_with_args: if variable doesn't exist, return empty string

### LINKED ISSUE #5378: Blog page showing a 500 error page
The following blog page is not loading for me. Instead, it is throwing a 500 internal server error: https://ubuntu.com/blog/minimal-ubuntu-released

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- `extends_with_args` tag can take template variables, added a check to make sure the variable exists, and if it doesn't, return an empty string instead of an error
- fixed grammar mistake on 500 page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/blog/minimal-ubuntu-released](http://0.0.0.0:8001/blog/minimal-ubuntu-released)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure that you see the post "Minimal Ubuntu, on public clouds and Docker Hub", and not a 500 error page

## Issue / Card

Fixes #5378 and #5379

### CODE REVIEW

File: webapp/templatetags/utils.py [hunk 1]
```diff
@@ -109,7 +109,11 @@ def render(self, context):
                 extra_context[key] = value[1:-1]
             else:
                 # It's a variable
-                extra_context[key] = template.Variable(value).resolve(context)
+                try:
+                    variable = template.Variable(value)
```
Comment (nottrobin) [inline webapp/templatetags/utils.py]: Why create this `variable` variable?

Comment (nottrobin) [reply webapp/templatetags/utils.py]: Oh 'cos the line is too long?

Comment (nottrobin) [reply webapp/templatetags/utils.py]: I would have done:

``` python3
extra_context[key] = template.Variable(value).resolve(
    context
)
```

Or let Black do it.

Comment (sowasred2012) [reply webapp/templatetags/utils.py]: To keep the linter happy, it flagged up the line length when using `extra_context[key] = template.Variable(value).resolve(context)` - do you see a better way?

**Pull request merged**